runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineResult.java

```diff
@@ -104,16 +104,15 @@ protected abstract State awaitTermination(Duration duration)
   @Override
   public State waitUntilFinish(final Duration duration) {
     try {
-      state = awaitTermination(duration);
+      State finishState = awaitTermination(duration);
```

Comment:
I don't like inlining here since `awaitTermination` is a big operation not a state getter. I think it should be expressed in a separate line.